tests/lac/vector_type_traits_is_serial_07.output

```diff
@@ -0,0 +1,4 @@
+
+DEAL::1
```

Comment:
Since this test is for `Trilinos` and with MPI types, the output file name must be adjusted to `vector_type_traits_is_serial_07.with_trilinos=true.with_mpi=true.output`